src/core/BufferAttribute.js

```diff
@@ -102,6 +102,16 @@ THREE.BufferAttribute.prototype = {
 
 	},
 
+	setRange: function ( index, typedArray ) {
```

Comment:
My misunderstanding, didn't realise TypedArray.set took regular arrays, even though I used same function as the other set function! 

Bit of confusion after tying to work out what copyAt was doing. What's your view on copyAt?